postgres/datadog_checks/postgres/postgres.py

```diff
@@ -136,6 +137,27 @@ def execute_query_raw(self, query):
             cursor.execute(query)
             return cursor.fetchall()
 
+    @property
+    def custom_queries(self):
+        if self._custom_queries is None:
```

Comment:
we're losing a fair amount of the validation and feedback from the old method `_collect_custom_queries`, like checking & logging for missing metric_prefix, query and column. Is that something we should retain?

Reply:
The metric prefix checking should no longer be needed because we actually _don't_ want users to supply that prefix anymore (and I should update the config as well now that you point that out).

But the handling of other input validation or errors are handled in QueryExecutor / QueryManager here: https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/utils/db/core.py. 

I think the checking is not as robust, but it is the same logic for all of the other DB integrations (MySQL, SQL Server, Oracle, etc.)